apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2535,10 +2576,22 @@ check_tx_and_verify_signatures(SignedTx, Updates, Mod, Data, Pubkeys, ErrTypeMsg
     ChannelPubkey = cur_channel_id(Data),
     ExpectedRound = next_round(Data),
     #data{state = State, opts = Opts, on_chain_id = ChannelPubkey} = Data,
+    Aetx = aetx_sign:tx(SignedTx),
     case aetx:specialize_callback(aetx_sign:innermost_tx(SignedTx)) of
         {aesc_settle_tx, _Tx} ->
             %% TODO: conduct more relevant checks
             verify_signatures_onchain_check(Pubkeys, SignedTx);
+        {aesc_client_reconnect_tx = Mod, Tx} ->
+            MyPubkey = my_account(Data),
+            MyRole = Data#data.role,
```

Comment:
Should be moved  upwards into line 2576.

Reply:
Done